addons/l10n_sa/data/account_data.xml

```diff
@@ -0,0 +1,8 @@
+<odoo>
+    <data noupdate="1">
+        <!-- set VAT label to show on invoice report -->
+        <record id="base.sa" model="res.country">
+            <field name="vat_label">VAT Number</field>
+        </record>
+    </data>
+</odoo>
```

Comment:
I don't really care, but some people are going to complain if you don't put the an empty line at the end of those files :p (multiple occurences; check them all)

Reply:
fixed